packages/compass-components/src/components/modal-footer-button.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { css } from '@leafygreen-ui/emotion';
+import type { ButtonProps } from '@leafygreen-ui/button';
+import { Button } from './leafygreen';
+import { Theme, useTheme, withTheme } from '../hooks/use-theme';
+
+const buttonStyle = {
+  [Theme.Light]: css({
+    margin: '0 2px',
+    '&:first-of-type': {
+      margin: '0 0 0 5px',
+    },
+    '&:last-of-type': {
+      margin: '0 5px 0 0',
+    },
+  }),
+  [Theme.Dark]: css({
+    margin: '0 2px',
+    '&:first-of-type': {
+      margin: '0 0 0 4px',
+    },
+    '&:last-of-type': {
+      margin: '0 4px 0 0',
```

Comment:
does the margin changing with the theme here?

Reply:
Yes that's copied from confirmation modal. It is all their CSS.